x-pack/plugins/security_solution/public/common/components/response_actions/endpoint_action_results.tsx

```diff
@@ -52,7 +56,7 @@ export const EndpointResponseActionResults = ({ action }: EndpointResponseAction
 
   return (
     <EuiComment
-      username={rule?.name}
+      username={ruleName && ruleName[0]}
```

Comment:
Q: Should we show multiple rule names here or the `ruleName` list always contains a single rule name?

Reply:
Thanks for raising this, ruleName always is an array of 1 rule. Just the way the response comes back to us. However I could get the first element earlier, so I am gonna do this. Thanks!